python/pyarrow/parquet/__init__.py

```diff
@@ -3419,7 +3419,7 @@ def read_metadata(where, memory_map=False, decryption_properties=None):
       num_rows: 3
       num_row_groups: 1
       format_version: 2.6
-      serialized_size: 561
+      serialized_size: 562
```

Comment:
Perhaps we should not care about the actual size in docstring examples.
```suggestion
      serialized_size: ...
```